src/gui/widgets/mapper_proxystyle.cpp

```diff
@@ -95,6 +95,16 @@ void MapperProxyStyle::drawPrimitive(QStyle::PrimitiveElement element, const QSt
 			drawSegmentedButton(segment, overridden_element, option, painter, widget);
 			return;
 		}
+		if (touch_mode && element == PE_PanelButtonTool)
+		{
+			auto const& palette = option->palette;
+			auto const& window = palette.window().color();
+			auto const fill = QBrush(qGray(window.rgb()) > 127 ? window.darker(125) : window.lighter(125));
+			painter->setPen(Qt::NoPen);
+			painter->setBrush(fill);
+			painter->drawRoundedRect(option->rect, 5.0, 5.0, Qt::AbsoluteSize);
+			return;
+		}
```

Comment:
For some reason, all tool modifier buttons appear as "active", i.e. dark grey, with this change. 
![image](https://user-images.githubusercontent.com/5584406/87668532-8598cb00-c76c-11ea-8f02-92e1a6149acf.png)

Maybe some other status information needs to be taken into account here.

Reply:
Quick&dirty fix for the issue:
```
--- a/src/gui/widgets/mapper_proxystyle.cpp
+++ b/src/gui/widgets/mapper_proxystyle.cpp
@@ -99,7 +99,7 @@ void MapperProxyStyle::drawPrimitive(QStyle::PrimitiveElement element, const QSt
                {
                        auto const& palette = option->palette;
                        auto const& window = palette.window().color();
-                       auto const fill = QBrush(qGray(window.rgb()) > 127 ? window.darker(125) : window.lighter(125));
+                       auto const fill = QBrush(bool(qGray(window.rgb()) > 127) ^ !option->state.testFlag(QStyle::State_On) ? window.darker(125) : window.lighter(125));
                        painter->setPen(Qt::NoPen);
                        painter->setBrush(fill);
                        painter->drawRoundedRect(option->rect, 5.0, 5.0, Qt::AbsoluteSize);

Reply:
The dark panel is now used only for `On` state, not for `Sunken` or `Raised`.